src/seedsigner/gui/components.py

```diff
@@ -432,18 +432,22 @@ def __post_init__(self):
             self.icon_font = Fonts.get_font(GUIConstants.ICON_FONT_NAME__FONT_AWESOME, self.icon_size, file_extension="otf")
         
         # Set width/height based on exact pixels that are rendered
-        (left, top, self.width, bottom) = self.icon_font.getbbox(self.icon_name, anchor="ls")
-        self.height = -1 * top
-
+        (self.left, self.top, right, bottom) = self.icon_font.getbbox(self.icon_name, anchor="ls")
+        self.width = -self.left + right
+        self.height = -self.top + bottom
```

Comment:
We'll need to test the icon vertical centering (e.g. the small right chevron for nested menu options). Icon centering behavior can be a little inconsistent since they can have different relationships with the baseline. I'm not sure I got them all to behave in a perfectly foolproof/consistent manner.

Reply:
Agree, it looks slightly too high now. I assume it was properly centered before. One possible solution would be to have the anchor as parameter, so f.e. for these kinds of icons one could use middle/middle aligntment (`mm`).

Reply:
That might help. Though I think it has to do with how the icon itself is defined. Like if a "-" is meant to float above the baseline, I'm not sure there's a way for us to repurpose it and get perfect vertical centering ourselves. In other words, "middle" for a dash might be halfway down the empty space below it, but "middle" for a bigger symbol is more likely to be where we'd expect it to be.